topics/transcriptomics/tutorials/mirna-target-finder/tutorial.md

```diff
@@ -0,0 +1,726 @@
+>
+> 1. {% tool [FastQC](toolshed.g2.bx.psu.edu/repos/devteam/fastqc/fastqc/0.72+galaxy1) %} with the following parameters:
+>    - {% icon param-collection %} *"Dataset collection"*: `Control mRNA`
+> 2. Rename the outputs as `FastQC unprocessed control mRNA: RawData` and `FastQC unprocessed control mRNA: Webpage`
+> 3. Repeat the previous step with the `BR treated mRNA` collection.
+> 4. {% tool [Merge Collections](__MERGE_COLLECTION__) %} with the following parameters:
+>    - In *"Input collections"*:
+>        - *"1.Input Collections"*: `FastQC unprocessed control mRNA: RawData`
+>        - *"2.Input Collections"*: `FastQC unprocessed BR treated mRNA: RawData`
+> 5. {% tool [MultiQC](toolshed.g2.bx.psu.edu/repos/iuc/multiqc/multiqc/1.8+galaxy1) %} with the following parameters:
+>    - In *"Results"*:
+>      - *"Which tool was used generate logs?"*: `FastQC`
+>      - {% icon param-collection %} *"Dataset collection"*: select the output generated in the previous step.
+>   - In *"Report title"*: `mRNA initial quality check`
+> 4. Click on the {% icon galaxy-eye %} (eye) icon and inspect the generated HTML file
+>
+{: .hands_on}
+
+> ### {% icon question %} Questions
+>
+> Is there any parameters that indicate the need to process the samples in order to improve its quality?
+>
+> > ### {% icon solution %} Solution
+> >
+> > All parameters are within accepted limits. However, the adapter content shows the presence of universal adapters in our readings, which can be removed to avoid possible interferences at later stages.
+> >
+> >    ![Experimental design](../../images/miRNA_adapter_01.png "Experimental design")
+> >
+> {: .solution}
+{: .question}
+
+> ### {% icon comment %} Comments
+> Although our samples have adapters, we are not going to trim them in this case. We will explain the reason in the next section.
+{: .comment}
+
+## Quantification of gene expression: **Salmon**
+
+After performing the quality assessment of the reads, we can move on to quantifying the gene expression. The aim of this step is to identify which transcript each read comes from and the total number of reads associated with each transcript. In this tutorial we will use the [Salmon](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5600148/) tool for the quantification of mRNA transcripts.
+
+![Experimental design](../../images/salmon_logo.png)
+
+One of the characteristics of __Salmon__ is that it doesn't require to perform a base-to-base alignment, which is the time-consuming step of tools such as STAR and HISAT2. Salmon relies in the quasi-mapping concept, a new mapping technique that allows the rapid and accurate mapping of RNA-seq reads to a target transcriptome. Rather than a standard alignment, quasi-mapping seeks to find the best mappings for each read, and does so by finding minimal collections of dynamically sized, right-maximal, matching contexts between target and query positions ({% cite Srivastava2016 %})
```

Comment:
```suggestion
One of the characteristics of __Salmon__ is that it doesn't require performing a base-to-base alignment, which is the time-consuming step of tools such as STAR and HISAT2. Salmon relies on the quasi-mapping concept, a new mapping technique that allows the rapid and accurate mapping of RNA-seq reads to a target transcriptome. Rather than a standard alignment, quasi-mapping seeks to find the best mappings for each read, and does so by finding minimal collections of dynamically sized, right-maximal, matching contexts between target and query positions ({% cite Srivastava2016 %})
```